handler/EventHandler.go

```diff
@@ -0,0 +1,209 @@
+package handler
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/ONSdigital/dp-import-reporter/config"
+
+	"github.com/ONSdigital/go-ns/log"
+	"github.com/coocood/freecache"
+)
+
+var failed = "failed"
```

Comment:
If this doesnt change it could probably be a const, though if it's only used in L96 where another string is being compared then maybe it doesn't need to be globally defined